### ISSUE TITLE
Add UI for forcefully logging out users

### ISSUE THREAD

**Issue by minrk** [jupyterhub/jupyterhub#706]
We should add a few UI actions in the admin panel and/or CLI for revoking tokens/cookies. You can do this by hand right now with a script accessing the db, but that's not exactly a great experience.

- Changing `user.cookie_id` logs out all browser sessions for that user
- Revoking API tokens for a user:

```python
for token in db.query(APIToken).filter(APIToken.user==user):
    db.delete(token)
```

**Comment by katsar0v** [comment]
Off-topic: do notebook containers shut down when a user/session is inactive/logged out

**Comment by willingc** [comment]
@katsar0v 
> Off-topic: do notebook containers shut down when a user/session is inactive/logged out?

Yes, logged out.

**Comment by minrk** [comment]
No, notebook containers are not shutdown until the user, admin, or external service like cull-idle shuts them down.

**Comment by SANJAY072000** [comment]
I am working on this issue
